index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  <title>Morrighan's Portfolio</title>
+</head>
+<body>
+   <!-- start toolbar -->
+  <header class="bg_white">
+    <div class="logo_placeholder">
+      <a href="index.html" class="logo xs-transparent_btn primary_b"
+        >My Logo</a>
+    </div>
+    <nav>
+      <a
+      href="#menu"
+      class="btn xs-transparent_btn btn_flex_center"
+      ><i class="material-icons primary_b">menu</i></a
+    >
+    </nav>
+  </header>
+  <!-- end toolbar -->
+
+  <!-- start main -->
+  <main class="shapes_bg bg_white">
+   <section class="headline">
+     <hgroup>
+       <h1 class="main_title neutral_b">I’m Radmila</h1>
+       <h2 class="main_title neutral_b">Glad to see you!</h2>
+     </hgroup>
+     <p class="descriptive_txt">
+       I’m a software developer! I can help you build a product , feature or
+       website Look through some of my work and experience! If you like what
+       you see and have a project you need coded, don’t hestiate to contact
+       me.
+     </p>
+     <p class="primary_n descriptive_txt fw_med upper">Let's connect</p>
+     <ul class="social">
+       <li>
+         <a href="#"
+           ><img src="icons/twitter.svg" alt="twitter logo icon"
+         /></a>
+       </li>
```

Comment:
### OPTIONAL
- [ ]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start).